pkg/aws/helpers.go

```diff
@@ -27,7 +27,23 @@ import (
 	rprtr "github.com/openshift/rosa/pkg/reporter"
 )
 
-var RoleNameRE = regexp.MustCompile(`^[\w+=,.@-]+$`)
+const RoleNameREString = `[\w+=,.@-]+`
+
+var RoleNameRE = regexp.MustCompile(fmt.Sprintf("^%s$", RoleNameREString))
+
+func getREOfAllAccRolesSuffixes() string {
```

Comment:
What does `REO` stands for?

Reply:
Regular Expression Of, the acronym is actually just `RE`. I think I saw this being used somewhere else in the code, that is why